Vagrantfile

```diff
@@ -37,9 +37,7 @@ fail 'Non-unique hostnames' if @network.collect { |_, v| v[:hostname] }.uniq!
 fail 'Non-unique ports' if @network.collect { |_, v| v[:ports].keys }.flat_map { |v| v }.uniq!
 
 def default_omnibus(config)
-  # https://github.com/cerner/cerner_splunk/issues/141
-  # config.omnibus.chef_version = :latest
-  config.omnibus.chef_version = '12.17.44'
```

Comment:
Do we need to keep the lock to take care of https://github.com/cerner/cerner_splunk/issues/142?